### PR TITLE
Add missing truncations from reset service and fix perf tests

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Builds on top of #5117 

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala [hunk 1]
```diff
@@ -1724,6 +1724,8 @@ private class JdbcLedgerDao(
         |truncate participant_command_completions cascade;
         |truncate participant_command_submissions cascade;
         |truncate participant_events cascade;
+        |truncate participant_event_flat_transaction_witnesses cascade;
```
Comment (gerolf-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala]: shouldn't this be taken care of by `truncate participant_events cascade;`?

Comment (stefanobaghino-da) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala]: I'd say yes, but I'd say I'll add a test right now to make sure that's the case (and to ensure we don't forget about this in the future).

**Comment by SamirTalwar-DA** [comment]
This looks sensible, but I think we're still missing some tables. Here's the list in my freshly-minted PostgreSQL database:

```
configuration_entries
contract_data
contract_divulgences
contract_key_maintainers
contract_keys
contract_observers
contract_signatories
contract_witnesses
contracts
disclosures
ledger_entries
package_entries
packages
parameters
participant_command_completions
participant_command_submissions
participant_event_flat_transaction_witnesses
participant_event_witnesses_complement
participant_events
parties
party_entries
```

And here's the list we're truncating as of this PR:

```
configuration_entries
contract_data
contract_key_maintainers
contract_keys
contract_witnesses
contracts
disclosures
ledger_entries
package_entries
parameters
participant_command_completions
participant_command_submissions
participant_event_flat_transaction_witnesses
participant_event_witnesses_complement
participant_events
```

We're missing:

```
contract_divulgences
contract_observers
contract_signatories
packages
parties
party_entries
```

Do we need to truncate any of them too?

**Comment by stefanobaghino-da** [comment]
> This looks sensible, but I think we're still missing some tables. Here's the list in my freshly-minted PostgreSQL database:
> 
> ```
> configuration_entries
> contract_data
> contract_divulgences
> contract_key_maintainers
> contract_keys
> contract_observers
> contract_signatories
> contract_witnesses
> contracts
> disclosures
> ledger_entries
> package_entries
> packages
> parameters
> participant_command_completions
> participant_command_submissions
> participant_event_flat_transaction_witnesses
> participant_event_witnesses_complement
> participant_events
> parties
> party_entries
> ```
> 
> And here's the list we're truncating as of this PR:
> 
> ```
> configuration_entries
> contract_data
> contract_key_maintainers
> contract_keys
> contract_witnesses
> contracts
> disclosures
> ledger_entries
> package_entries
> parameters
> participant_command_completions
> participant_command_submissions
> participant_event_flat_transaction_witnesses
> participant_event_witnesses_complement
> participant_events
> ```
> 
> We're missing:
> 
> ```
> contract_divulgences
> contract_observers
> contract_signatories
> packages
> parties
> party_entries
> ```
> 
> Do we need to truncate any of them too?

I'll keep this as is to make the minimum set of adjustments to make perf tests work, this is now tracked in #5130.

**Pull request merged**